backend/app/Http/Controllers/Admin/AdminQuizController.php

```diff
@@ -119,4 +119,48 @@ public function storeAdminQuizQuestion(Request $request, Quiz $quiz)
 
         return response($response, 201);
     }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+     public function showAdminQuizQuestions(Quiz $quiz)
+    {
+        return Question::where('quiz_id', $quiz->id)->simplePaginate(5);
+    }
+    
+    /**
+     * Remove the specified resource from storage.
+     *
+     * @param  int  $id
+     * @return \Illuminate\Http\Response
+     */
+     public function deleteAdminQuizQuestion(Question $question)
+    {
+        $question->delete();
+        
+        return $question;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+     public function showAdminQuizQuestion(Question $question)
+    {
+        $choices = Choice::where('question_id', $question->id)->get();
+
+        $question = [
+            'id' => $question->id,
+            'quiz_id' => $question->quiz_id,
+            'word' => $question->word,
+            'created_at' => $question->created_at,
+            'updated_at' => $question->updated_at,
+            'choices' => $choices,
+        ];
+
+        return $question;
```

Comment:
There is a shorter way of doing this.
```suggestion
return $question->with('choices')->get();
```

Reply:
This is called eagerloading. Mag agad ni iyang query based sa imong model function.
So dapat naa kay something like this sa imong `Question.php`
```
public function choices()
{
    return $this->hasHasMany(Choice::class);
}
```